core/src/main/resources/jenkins/model/Messages_de.properties

```diff
@@ -21,8 +21,8 @@
 # THE SOFTWARE.
 
 Hudson.BadPortNumber=Falsche Portnummmer {0}
-Hudson.Computer.Caption=Master
-Hudson.Computer.DisplayName=master
+Hudson.Computer.Caption=Built-in Knote
+Hudson.Computer.DisplayName=Built-in
```

Comment:
```suggestion
Hudson.Computer.Caption=Eingebauter Knoten
Hudson.Computer.DisplayName=Eingebauter Knoten
```

Reply:
I think the display name is what's shown in the side panel? In English it works with just "built-in" (IMO), less sure about German.

My first thought was to use "integriert" for the built-in equivalent, but that seems wrong without a noun too.

"Interner Knoten" and "Intern" could work?

Reply:
`Hudson.Computer.Caption` is used in the `/computer/` overview, "Interner Knoten" sounds fine too.

Reply:
> "Interner Knoten" and "Intern" could work?

I already merged the original suggestion, but if someone offers this new suggestion in the GitHub UI I can accept it as well. I don't have a preference about the specific translation, just that it matches the logical structure of the original (with two different strings).